### ISSUE TITLE
Add ability to set weak password

### ISSUE THREAD

**Issue by RoboBurned** [linuxmint/cinnamon#9291]
```
 * Cinnamon version: Cinnamon 4.4.8
 * Distribution - Mint 19.3
 * VirtualBox
 * 64 bit
 ```

**Issue**
During installation of Mint 19.3 user is free in choosing his password. It can set it to any value even "1".  After installation I want to to change it to "123". Both "Account Details" and "Users and Groups" dialogs do not allow to change the password to "123" because it is too short.

I use VirtualBox to play with different Linux distibutives and I don't care about security at this point. I want to all my VMs have the same password. There are may be many reasons why user have to use short password. We should not limit it here.

**Steps to reproduce**
Install Linux Mint. During installation set any short password.
Log into the system and try to change password to any value shorter than 8 chrs by using either "Account Details" or "Users and Groups" dialog.
"Change" button is disabled.

**Expected behavior**
User is able to set any password. A warning text is displayed informing about password weakness.

**Other information**
Another more complex solution could be defining a password policy configured by administrator. I don't need this.

**Comment by PlazmaKG** [comment]
I imagine that this is a security thing. Having a password that short, and only numbers, breaks the purpose of a password in the first place, since that could be vert easily guessed, through brute force methods.

**Comment by RoboBurned** [comment]
@PlazmaKG I agree that short password can easily be break. All users should think of their security. I just don't like the method we try to enforce this.
For regular home user who is not a professional in Linux current approach does not allow user to change less secure password "1" to a more secure password "zx_@#C5" phrase and will use the one set during installation. This does not bring security.
For professional user this is not an issue, he can use sudo passwd to change it to any value, but I think UI and all "Settings" applets were developed to make it easier to change settings without using command line. Forcing such users use command line does not bring security as well.
We cannot require user to be security expect. We just should guide him, suggest better practices and warm about potentially risky situations without limiting him in reaching his goals.
